x-pack/plugins/index_lifecycle_management/public/application/sections/policy_list/components/policy_table.tsx

```diff
@@ -69,6 +69,18 @@ const managedPolicyTooltips = {
   ),
 };
 
+const deprecatedPolicyTooltips = {
+  badge: i18n.translate('xpack.indexLifecycleMgmt.policyTable.templateBadgeType.deprecatedLabel', {
+    defaultMessage: 'Deprecated',
+  }),
+  badgeTooltip: i18n.translate(
+    'xpack.indexLifecycleMgmt.policyTable.templateBadgeType.deprecatedDescription',
+    {
+      defaultMessage: 'This policy is deprecated and should not be relied on.',
```

Comment:
```suggestion
      defaultMessage: 'This policy is no longer supported and might be removed in a future release. Instead, use one of the other policies available.',
```
Same as previous suggestion